packages/web3-shared/base/src/specs/index.ts

```diff
@@ -94,28 +94,23 @@ export enum SourceType {
     R2D2 = 'R2D2',
 }
 
-export enum SearchSourceType {
-    Main = 'Main',
-    Sidebar = 'Sidebar',
-}
-
-/** @deprecated use SearchSourceType stead */
+/** @deprecated use SearchResultType stead */
 export enum SearchKeywordType {
     Address = 'Address',
     Domain = 'Domain',
 }
 
 export enum SearchResultType {
-    // e.g., 0xd8da6bf26964af9d7eed9e03e53415d37aa96045
-    Address = 'Address',
+    // e.g., 0x6a7122B831c2B79c508A978f73f2ee23171279B3
+    EOA = 'EOA',
     // e.g., vitalik.eth or vitalik.bnb
     Domain = 'Domain',
     // e.g., $MASK #MASK
-    FungibleToken = 'FungibleToken',
-    // e.g., $PUNK #PUNK
-    NonFungibleToken = 'NonFungibleToken',
-    // e.g., $PUNK #PUNK
-    NonFungibleCollection = 'NonFungibleCollection',
+    TrendingTokenByKeyword = 'TrendingTokenByKeyword',
+    // e.g., 0x69af81e73a73b40adf4f3d4223cd9b1ece623074 (Mask Network)
+    TrendingTokenByAddress = 'TrendingTokenByAddress',
```

Comment:
They are the same thing `FungibleToken`. And please don't remove `NonFungibleToken`. It's used in the next feature.